src/test/lib/MtuTest.cpp

```diff
@@ -45,6 +49,12 @@ struct ResetSettings {
 void
 QuicTestMtuSettings()
```

Comment:
This test is really fragile. It assumes a max MTU that we might change in the future...